README.md

```diff
@@ -1,5 +1,24 @@
 # ENEE408I_Fall_2020
-Shared repository for all students in ENEE408I_Fall_2020 offering
+Shared repository for students in the ENEE408I Fall 2020 offering
+
+## Usage
+Each team should have a folder team1, team2, etc.
+
+[This tutorial](https://www.neonscience.org/git-setup-remote) is good for learning how to maintain your Git repo.
+**Note 1:** You do not need to fork this repo (since you were given write access).
+**Note 2:** After cloning should create, checkout, and work on your own branch instead of main. See the [Git Command Cheatsheet](http://rogerdudler.github.io/git-guide/files/git_cheat_sheet.pdf).
+**Note 3:** The `gitk` command will easily visualize the git tree. `gitk --all` will show you an even more detailed tree. Install with `sudo apt install gitk`.
+**Note 4:** When you are ready to share your work with the class through the `main` branch, a good way is to make a Pull Request. For example this file was commited with Pull Request #1. You can then comment on the pull request and have a discussion.
+
+Remember sharing your code with teammates is encouraged! If setup right your teammates can even import code from another teams folder instead of copy-pasting it.
+
+## Contents
+
+* Examples - Short pieces of code and examples of software the class finds useful
+* Notes - Short notes on how to install or fix issues with pieces of software
+* team1
```

Comment:
This is an example of commenting on some code in a pull request. You could tell the author they made a mistake here.